### DISCUSSION TITLE
Device present under custom_components/tuya_local/devices

### DISCUSSION THREAD

**Discussion by subexx** [make-all/tuya-local#945] [Q&A] [answered]
Hi, I have a Kogan Smart Bidet. I can see it under the custom_components/tuya_local/devices but when i try to add it to HA it says device is not supported. Any idea how to fix it?
![image](https://github.com/make-all/tuya-local/assets/91825280/e166f7af-4cff-4046-ae51-23d7fae5c1f3)
![image](https://github.com/make-all/tuya-local/assets/91825280/4dc6e3eb-3c0e-49fc-897e-1ed49e222f36)

**Comment by make-all** [comment, marked as answer]
The issue has been fixed under issue #559
